.github/workflows/python-wheels-publish.yml

```diff
@@ -39,6 +39,8 @@ jobs:
       - name: Create sdist
         # Only create it once.
         if: ${{ matrix.os == 'ubuntu-latest' }}
+        env:
+          OPENEXR_RELEASE_CANDIDATE_TAG: ${{ github.ref_name }}
```

Comment:
Shouldn't this be set only for release candidates?

Reply:
Good point, I removed it. Technically it doesn't matter since the result would be the same, since the release tag will be the same as the version, but logically the `RELEASE_CANDIDATE_TAG` should not play a part here. Thanks!